tests/scripts/task_config_build_gpu.sh

```diff
@@ -30,7 +30,7 @@ echo set\(USE_VULKAN ON\) >> config.cmake
 echo set\(USE_OPENGL ON\) >> config.cmake
 echo set\(USE_MICRO ON\) >> config.cmake
 echo set\(USE_MICRO_STANDALONE_RUNTIME ON\) >> config.cmake
-echo set\(USE_LLVM llvm-config-9\) >> config.cmake
+echo set\(USE_LLVM \"/usr/bin/llvm-config-9 --link-static\"\) >> config.cmake
```

Comment:
just curious, why do we need to link static LLVM?

Reply:
oh i understand now from this thread: https://github.com/apache/tvm/issues/9362#issuecomment-955263494